windows-server-container-tools/New-ContainerHost/New-ContainerHost.ps1

```diff
@@ -89,7 +89,7 @@ param(
     [Parameter(ParameterSetName="Staging", Mandatory=$true)]
     [string]
     [ValidateNotNullOrEmpty()]
-    $VhdPath = "http://aka.ms/containerhostvhd",
+    $VhdPath = "https://aka.ms/containerhostvhd",
```

Comment:
This URL is redirected back to an http URL http://download.microsoft.com/download/...